frontend/js/helpers/Transaction.js

```diff
@@ -0,0 +1,131 @@
+//common functions that can be used in different cases
+
+
+async function Get(path) {
+    return await ApiRequest("http://localhost:3000/" + path, "GET");
+}
+
+async function Post(path, body) {
+    return await ApiRequest("http://localhost:3000/" + path, "POST", body);
+}
+
+async function ApiRequest(url, type, body) {
+    return await $.ajax({
+        url: url,
+        type: type,
+        data: JSON.stringify(body),
+        contentType: "application/json",
+        success: (data) => {
+            return data;
+        },
+        error: (err) => {
+            console.log(err);
+        },
+    });
+}
+
+
+//2023.8.25 
+$(document).ready(async () => {
+  
+  //1. show-hide from,to,id depending on type
+  
+  const accountId=$('#accountId');
+  const accountIdFromField = $('#accountIdFrom');
+  const accountIdToField = $('#accountIdTo');
+  //accountIdFrom &accountIdTo is hidden as default
+  accountIdFromField.hide();
+  accountIdToField.hide();
+  $('input[name="transactionType"]').change(() => {
```

Comment:
This input element has its own id.
How about select this element by using its id?